src/components/Header/Header.jsx

```diff
@@ -0,0 +1,255 @@
+                                <MobileHamburgerLine />
+                            </MobileHamburgerContainer>
+                        </ContainerHeder>
+                        <MobileHamburgerbackground style={{ transform: isOpen ? "translateX(0%)" : "translateX(120%)", opacity: isOpen ? "1" : "0.3" }}>
+                            <MobileHamburgerLoginBox>
+                                <MenuHeaderBlock>
+                                    <MenuInitiaLetterCircle>
+                                        <MenuInitiaLetter>{nameLetter}</MenuInitiaLetter>
+                                    </MenuInitiaLetterCircle>
+                                    <MenuName>{UserName}</MenuName>
+                                    <NavLink to="login" onClick={`${toggle}${logOut}`}>
+                                        <ExitIconMobile />
+                                    </NavLink>
+                                </MenuHeaderBlock>
+                            </MobileHamburgerLoginBox>
+                            <MobileHamburgerCrossBox onClick={toggle}>
+                                <MobileHamburgerCross>
+                                    <MobileHamburgerCross1 />
+                                </MobileHamburgerCross>
+                            </MobileHamburgerCrossBox>
+
+                            <NavLink to="home" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenuUp>Главная</MobileHamburgerSectionMenuUp>
+                            </NavLink>
+                            <NavLink to="planning" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Планирование</MobileHamburgerSectionMenu>
+                            </NavLink>
+                            <NavLink to="awards" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Награды</MobileHamburgerSectionMenu>
+                            </NavLink>
+                            <NavLink to="contacts" onClick={toggle}
+                                style={({ isActive }) => { return { color: isActive ? 'black' : 'white' } }}>
+                                <MobileHamburgerSectionMenu>Контакты</MobileHamburgerSectionMenu>
+                            </NavLink>
+                        </MobileHamburgerbackground>
+                    </>
+                    }
+                </>
+            }
+        </>
+    )
+}
+export default Header
+
+const LogoHead = () => (
+    <NavLink to="home">
+        <LogoHeader>
+            KidsLike
+            <Logo />
+        </LogoHeader>
+    </NavLink>
+)
+const MenuAuthDeskTab = () => (
```

Comment:
If you decided to make this code a separate component it could be placed into separete file

Reply:
Can I create a file called Helper and add components like this to that file?)